app/models/student.rb

```diff
@@ -6,13 +6,15 @@ class Student < ApplicationRecord
   belongs_to :team, optional: true
   belongs_to :user, optional: true
   has_and_belongs_to_many :parents
-  has_many :team_dues
+  has_many :events, through: :services
```

Comment:
![tmerror1](https://user-images.githubusercontent.com/10386184/34342633-1528d75c-e984-11e7-87a1-60f12eb01c2b.png)

This association has to be defined below
```ruby
has_many :services
```